Source/WebCore/PAL/pal/crypto/commoncrypto/CryptoDigestCommonCrypto.cpp

```diff
@@ -190,34 +175,32 @@ String CryptoDigest::toHexString()
     return String::fromUTF8(result.span());
 }
 
-std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, const Vector<uint8_t>& data, UseCryptoKit useCryptoKit)
+Vector<uint8_t> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, std::span<const uint8_t> data)
```

Comment:
Changing from a Vector reference to a span can have lifetime implications at the call site, right? Because it makes it possible for someone to do something like:
```
Vector<uint8_t> my_data = ...;
auto my_span = my_data.span();
dispatch_async(dispatch_get_main_queue(), ^{
    sha256(my_span); // UAF
});
```

I'm not sure you need to change it back per se, but wanted to get your opinion on whether this is the right abstraction.